README.md

```diff
@@ -1,2 +1,75 @@
 # Countries
 This app displays exciting information and facts from countries all over the world
+
+## Built With
+
+- REACT.JS
+- REDUX
+- TAILWIND-CSS
+
+## Install
+### Run the project in your local machine
+
+- Clone the repository
+```
+  $ git clone https://github.com/Estherstarr/countriesAPI.git
+  $ cd countriesAPI
+```
+
+- Install all dependancies
+```
+$ npm install
+```
+
+- Run the server
+```
+$ npm start
+```
+- Open your browser and go to [localhost:3000](http://localhost:3000/)
+
+
+### Run Linters tests
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+$ npm install
+```
+Track HTML linter errors run:
+```
+$ npx hint .
+```
+Track CSS linter errors run:
+```
+$ npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+$ npx eslint .
+```
+
+### Deployment
+- GitHub pages
+
+## Author
+
+👤 **Author : ONWUANIBE ONOMEH**
+
+- GitHub: [@githubhandle](https://github.com/Estherstarr/)
+- Twitter: [@twitterhandle](https://twitter.com/AnibeEsther)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Let's add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] Let's add `MIT.md` file for licence. now it is pointing to non existing file